modules/installing-oci-about-agent-based-installer.adoc

```diff
@@ -36,5 +36,5 @@ To ensure the best performance conditions for your cluster workloads that operat
 * Basic environment: 500 GB, and 60 VPUs.
 * Heavy production environment: More than 500 GB, and 100 or more VPUs.
 
-Consider reserving additional VPUs to provide sufficient capacity for updates and scaling activities. For more information about VPUs, see Volume Performance Units in the Oracle documentation.
+Consider reserving additional VPUs to provide sufficient capacity for updates and scaling activities. For more information about VPUs, see Volume Performance Units in the _Oracle_ documentation.
```

Comment:
> For more information about VPUs, see Volume Performance Units in the _Oracle_ documentation.

I see this is existing text. But, we don't have a link?

Reply:
```suggestion
Consider reserving additional VPUs to provide sufficient capacity for updates and scaling activities. For more information about VPUs, see Volume Performance Units in the Oracle documentation.
```

Reply:
OK as per recent guideline updates https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-xref-attribute